package.json

```diff
@@ -97,22 +97,23 @@
     "@types/jest": "26.0.23",
     "@types/react": "17.0.9",
     "@types/react-helmet": "6.1.1",
-    "@types/styled-components": "5.1.9",
     "@types/react-sticky-el": "^1.0.2",
+    "@types/styled-components": "5.1.9",
     "gatsby-remark-sectionize": "1.0.0",
     "husky": "7.0.1",
     "jest": "27.0.3",
     "jest-dom": "4.0.0",
     "lint-staged": "^11.0.0",
     "loader-utils": "2.0.0",
     "prettier": "2.3.2",
-    "prisma": "2.24.0",
+    "prettier-plugin-prisma": "2.28.0",
```

Comment:
Does this depend on a peer dependency `prisma` or does it bring its own?

Reply:
@janpio it has it's own wasm binding of the `datamodel` from [prisma-engines](https://github.com/prisma/prisma-engines) 

https://unpkg.com/browse/prettier-plugin-prisma@2.28.0/wasm/